### PR TITLE
add openshift/kubevirt-csi-driver-operator repo

### PR DESCRIPTION
**Pull request by ydayagi**
https://github.com/openshift/kubevirt-csi-driver-operator
Signed-off-by: Yaron Dayagi <ydayagi@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift/kubevirt-csi-driver-operator/openshift-kubevirt-csi-driver-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+base_images:
+  base:
+    name: "4.8"
+    namespace: ocp
+    tag: base
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```
Comment (rgolangh) [inline ci-operator/config/openshift/kubevirt-csi-driver-operator/openshift-kubevirt-csi-driver-operator-master.yaml:9]: need to change to go 1.15 - this is what's used for the project https://github.com/openshift/kubevirt-csi-driver/blob/master/go.mod

Comment (rgolangh) [reply ci-operator/config/openshift/kubevirt-csi-driver-operator/openshift-kubevirt-csi-driver-operator-master.yaml:9]: sorry my mistake, this is the operator

Comment (ydayagi) [reply ci-operator/config/openshift/kubevirt-csi-driver-operator/openshift-kubevirt-csi-driver-operator-master.yaml:9]: operator uses 1.13 in voth go.mod and Dockerfile

**Pull request closed without merging**